src/app/ipv/middleware.js

```diff
@@ -104,7 +107,7 @@ async function handleBackendResponse(req, res, backendResponse) {
 
   if (
     backendResponse?.client &&
-    tryValidateClientResponse(backendResponse.client)
+    validateRedirectUrlInResponse(backendResponse.client, true)
```

Comment:
Hmm I'm not sure about this - not a fan of boolean flag args but maybe the function could just take a `response` (object of any shape) and check for the presence of `redirectUrl` in that object. But then we lose the specific error message (client vs CRI) so I'm not convinced this refactor is really worth it 😕

Reply:
Yes, we have a same object for both responses and we can't differentiate.

Reply:
Removed it, thanks